training/tasks/train.js

```diff
@@ -50,6 +51,13 @@ module.exports = async function main(task, argv) {
     const genieJob = Genie.Training.createJob(options);
     genieJob.on('progress', (value) => {
         task.setProgress(value);
+	if (Config.TENSORBOARD_DIR) {
+            cmd.exec('aws', [ 's3', 'sync',
+                 workdir, `${Config.TENSORBOARD_DIR}/${task.jobId}/${task.info.model_tag}:${task.language}/`,
+                 '--exclude="*"',
+                 '--include="*tfevents*"'
+            ]);
```

Comment:
This should probably be `AbstractFS.sync`, to avoid hardcoding s3. We can add other options for include/exclude (rsync, used by the local backend, supports them too)

Reply:
Done. I've added an extraArgs to sync and upload .